### PR TITLE
Handles SOAP result null output

### PR DESCRIPTION
**Pull request by simonelbaz**

### CODE REVIEW

File: lib/client.js [hunk 1]
```diff
@@ -260,7 +260,7 @@ Client.prototype._invoke = function(method, args, location, callback, options, e
       } catch (error) {
         //  When the output element cannot be looked up in the wsdl and the body is JSON
         //  instead of sending the error, we pass the body in the response.
-        if(!output.$lookupTypes) {
+        if((output !== null) && (!output.$lookupTypes)) {
```
Comment (jsdevel) [inline lib/client.js]: Try `if (!output || !output.$lookupTypes) {`

**Comment by simonelbaz** [comment]
ok

**Comment by jsdevel** [comment]
Thanks for the contributtion!

**Comment by simonelbaz** [comment]
Hi jsdevel,
that's ok. Is it possible to be added to contributors, even it is a minor contribution ?
Thanks

**Comment by jsdevel** [comment]
I believe that's handled [here](https://github.com/vpulim/node-soap/graphs/contributors).

**Pull request merged**